lib/src/model/directives/hide_constant_implementations.dart

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2023, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:dartdoc/src/model/documentation_comment.dart';
+import 'package:dartdoc/src/model/model_element.dart';
+
+final _hideConstantImplementationsRegExp =
+    RegExp(r'{@hideConstantImplementations}');
+
+/// Implement parsing the hideConstantImplementations dartdoc directive
+/// for this [ModelElement].  Used by [Container].
+mixin HideConstantImplementations on DocumentationComment {
+  bool? _hideConstantImplementations;
+
+  /// [true] if the {@hideConstantImplementations} dartdoc directive is present
+  /// in the documentation for this class.
+  bool get hideConstantImplementations {
```

Comment:
The name here makes it sound like a function, like calling `hideConstantImplementations` will do something or set something. What about `hasHideConstantImplementations`, like how the analyzer APIs are named?

https://dart.dev/guides/language/effective-dart/design#prefer-a-non-imperative-verb-phrase-for-a-boolean-property-or-variable